src/MicrosoftTeams.ts

```diff
@@ -486,8 +486,8 @@ namespace microsoftTeams {
             removeHandler = handler;
         }
 
-        function handleSave(): void {
-            let saveEvent = new SaveEventImpl();
+        function handleSave(result: SaveResult): void {
```

Comment:
The result parameter should be marked as optional since it won't always be provided (e.g. in the case of tabs).